src/DittoMachine.sol

```diff
@@ -271,9 +273,10 @@ contract DittoMachine is ERC721, ERC721TokenReceiver, ERC1155TokenReceiver, Clon
             CloneShape memory cloneShape = cloneIdToShape[cloneId];
             uint256 heat = cloneShape.heat;
 
-            uint256 minAmount = _getMinAmount(cloneShape);
+            uint256 feeRefund = _getBlockRefund(cloneId); // check if bids have occured within the current block
+            uint256 minAmount = _getMinAmount(cloneShape, feeRefund != 0);
             // calculate subsidy and worth values
-            uint256 subsidy = minAmount * (MIN_FEE * (1 + heat)) / DNOM;
+            uint256 subsidy = (_amount * (MIN_FEE * ((feeRefund == 0 ? 1 : 0) + heat))) / DNOM;
```

Comment:
why the change from `minAmount` to `_amount`?

Reply:
made a quick explanation in [this comment](https://github.com/ditto-lab/ditto/pull/51#issuecomment-1224575219). 
Basically the fees should be applied to the total amount transferred to the protocol, so the new worth of the clone rather than the old worth of the clone. when the clone is minted the fees are based on the amount transferred, and i think the behaviour should be consistent in the `else` block as well.